credentials/alts/core/common.go

```diff
@@ -18,6 +18,7 @@
 
 // Package core contains common core functionality for ALTS.
 // Disclaimer: users should NEVER reference this package directly.
+//go:generate ./regenerate.sh
```

Comment:
Move this above the pacakge comment to be consistent with other `go generate`s?

And I'm curious if this will be rendered in godoc if we leave it this way...

Reply:
Done.  Also, yes it would have shown up :/